etna/search/templatetags/search_tags.py

```diff
@@ -218,3 +218,12 @@ def search_title(search_tab) -> str:
     elif search_tab == SearchTabs.WEBSITE.value:
         label = "Website search results"
     return label
+
+
+@register.simple_tag
+def extended_in_operator(lhs_operand, *rhs_operand_list) -> bool:
+    """
+    Input params are template tags
+    Returns True when rhs_operand_list contains lhs_operand value, False otherwise
+    """
+    return (lhs_operand in rhs_operand_list) or False
```

Comment:
This is not so readable for me unless I read the comments, I suggest by default the function should return true , 
something like that 
if ((lhs_operand not in rhs_operand_list)
return false
return true

Reply:
Its **native** to Python and a way we can write **powerful and elegant Python one-liners,**
the evaluation is either (True or False) - short-circuit evaluation for OR here is True - **Advantage here over other constructs**.
or (False or False)

Reply:
The code you have written will loop through the list and if it finds the element then it uses the OR operator  True OR False ==True  return True or if not then False OR False==Fasle , in fact, you are adding further overhead, Why do you need to do this ?
Just make it simple:
if ((lhs_operand not in rhs_operand_list)
return false

Reply:
> I think code performance and simplicity are vital, could you please add unit tests, considering all possible outcomes, particularly edge cases like None, empty lists, and exceptions,...

Its easier to understand responses when its in context. I think  your mention _code performance and simplicity are vital_, has nothing to do with this bit, please see comments above on _short-circuit evaluation_ and _pythonic ways_. Why are you mentioning it in this context?

> I have noticed there are fewer unit tests recently

Again, Its easier to understand responses when its in context. Why are you mentioning the above in this context?

> I also encourage TTD as an approach. TDD is common for all programming languages :)

Good to discuss in dev catch-up.  Can you please give two examples where you have deployed TDD in this code base?

> could you please add unit tests, considering all possible outcomes, particularly edge cases like None, empty lists, and exceptions,...

I've added test cases.

Reply:
Pythonic ways - in a sense does show simplicity and also handles performance bit. We can discuss this in dev catch-up along with TDD. Hope this will be fine.

Reply:
Marking as resolved, if unresolved, please respond to above queries to your responses.